src/Razor/src/Microsoft.VisualStudio.RazorExtension/SyntaxVisualizer/SyntaxVisualizerControl.xaml.cs

```diff
@@ -141,6 +178,39 @@ public void ShowGeneratedCode()
             return;
         }
 
+        EnsureInitialized();
+
+        if (_languageServerFeatureOptions.UseRazorCohostServer)
+        {
+            if (_fileUriProvider.TryGet(_activeWpfTextView.TextBuffer, out var hostDocumentUri))
+            {
+                var request = new DocumentContentsRequest
+                {
+                    TextDocument = new TextDocumentIdentifier { DocumentUri = hostDocumentUri },
+                    Kind = GeneratedDocumentKind.CSharp
+                };
+
+                var response = _joinableTaskFactory.Run(async () =>
+                {
+                    var lspResponse = await _lspRequestInvoker.ReinvokeRequestOnServerAsync<DocumentContentsRequest, DocumentContentsResponse>(
+                        _activeWpfTextView.TextBuffer,
+                        "razor/generatedDocumentContents",
+                        RazorLSPConstants.RoslynLanguageServerName,
+                        request,
+                        CancellationToken.None);
+
+                    return lspResponse?.Response;
+                });
+
+                if (response != null)
+                {
+                    OpenGeneratedCode(response.FilePath, response.Contents);
+                    return;
+                }
```

Comment:
Replace this code with another call to the new ShowGeneratedCode_Cohost method I mentioned above.